dashboard/src/pages/CreateEventForm.vue

```diff
@@ -4,84 +4,120 @@
-          />
-          <FormControl
-            v-model="temp_event.event_permalink"
-            :type="'text'"
-            size="md"
-            label="Event Permalink"
-            description="Only enter the event endpoint. Events will be rendered at event/<event_permalink>"
-          />
-          <FormControl
-            v-model="temp_event.status"
-            :type="'select'"
-            :options="[
-              {
-                label: 'Draft',
-                value: 'Draft',
-              },
-              {
-                label: 'Live',
-                value: 'Live',
-              },
-              {
-                label: 'Concluded',
-                value: 'Concluded',
-              },
-              {
-                label: 'Cancelled',
-                value: 'Cancelled',
-              },
-            ]"
-            size="md"
-            label="Event Status"
-            description="Current status of the event."
-          />
-          <FormControl
-            v-model="temp_event.event_type"
-            :type="'select'"
-            :options="eventTypeOptions.data"
-            size="md"
-            label="Event Type"
-          />
-          <TextEditor
-            label="Event Description"
-            class="col-span-2"
-            placeholder="Write an event description"
-            :model-value="temp_event.event_description"
-            @update:model-value="($event) => (temp_event.event_description = $event)"
-          />
+      <div class="flex flex-col gap-5 my-6">
+        <div class="space-y-2">
+          <div class="prose">
+            <h3>Details</h3>
+          </div>
+          <div class="my-1 grid sm:grid-cols-1 md:grid-cols-2 gap-x-4 gap-y-6">
+            <FormControl
+              v-model="temp_event.event_type"
+              :type="'select'"
+              :options="eventTypeOptions.data"
+              size="md"
+              label="Event Type&ast;"
+            />
+            <FormControl
+              v-model="temp_event.event_name"
+              :type="'text'"
+              size="md"
+              label="Event Name&ast;"
+            />
+            <FormControl
+              v-model="temp_event.event_permalink"
+              :type="'text'"
+              size="md"
+              label="Event Permalink&ast;"
+            />
+            <FormControl
+              :value="getEventLink()"
+              size="md"
+              label="Event Link"
+              :disabled="true"
+              description="The event link will be as shown above."
+            />
+            <FormControl
+              v-model="temp_event.status"
+              :type="'select'"
+              :options="[
+                {
+                  label: 'Draft',
+                  value: 'Draft',
+                },
+                {
+                  label: 'Live',
+                  value: 'Live',
+                },
+                {
+                  label: 'Concluded',
```

Comment:
I think we needn't include Concluded and Cancelled in a create event form. They are CTAs that come after an event is made. Draft and Live makes sense

Reply:
makes sense, changing